src/get-started/java-binding.md

```diff
@@ -21,19 +21,23 @@ To run a node with your Java service you need to use Exonum Java application.
 
 There are several installation options:
 
-- [Manual installation](#manual-installation) — available for Mac OS and Linux
+- [Manual installation](#manual-installation) — available for Mac OS and Linux,
+  _recommended for Linux users_
 - [Homebrew package](#homebrew-package) — available for Mac OS only,
-  recommended for Mac users
+  _recommended for Mac users_
 - [Build from source](#build-from-source) — available for Mac OS and Linux
 
+Windows is not supported by Java Binding at the moment, consider using
+[WSL] or [Docker][docker-for-windows].
```

Comment:
WSL link is broken

Reply:
Not really, see [rendered](https://github.com/exonum/exonum-doc/blob/facf574152438fa68273175336bde76691288240/src/get-started/java-binding.md)